pkg/common/gatewayapi_utils_test.go

```diff
@@ -612,3 +613,111 @@ func TestGatewayWrapperPolicyRefsAnnotation(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func TestTargetHostnames(t *testing.T) {
+	testRoute := &gatewayapiv1alpha2.HTTPRoute{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "gateway.networking.k8s.io/v1alpha2",
+			Kind:       "HTTPRoute",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "test-route",
+			Namespace: "test-namespace",
+		},
+		Spec: gatewayapiv1alpha2.HTTPRouteSpec{
+			Hostnames: []gatewayapiv1alpha2.Hostname{"*.example.com", "*.x.com"},
```

Comment:
Damn! I'm sorry @didierofrivia my comment was unclear I realize now... the ones that _exists_ are x.com, q.com and z.com... all other letters are good candidates, but q, x and z are not 🤦